go.mod

```diff
@@ -1,12 +1,76 @@
 module github.com/hashicorp/vault-plugin-auth-oci
 
-go 1.12
+go 1.19
```

Comment:
Any reason we didn't bump to 1.21?

Reply:
No strong reason, mostly wanted to match the version in`.go-version`.